### ISSUE TITLE
PowerPoint web taskpane add-in: goToById doesn't work when presenting

### ISSUE THREAD

**Issue by gordonmleigh** [OfficeDev/office-js#924]
> Message from office-js bot:
We’re closing this issue because it has been inactive for a long time. We’re doing this to keep the issues list manageable and useful for everyone. If this issue is still relevant for you, please create a new issue.
> Thank you for your understanding and continued feedback.

When the slideshow has started, `goToById` doesn't change the current slide for a **taskpane** add-in on web.

This is a resubmission of #875 which @donlvMSFT closed despite me providing a minimal working reproduction repository and him confirming the repro and filing internal bug 3809617. He provided a content add-in manifest which appears to work as expected, but this issue is related to a taskpane add-in. He didn't provide the full code to me so I cannot investigate further myself. In any case I have identified a bug, provided detailed information for you to diagnose, and you have confirmed it so I'm pretty frustrated that the issue was closed anyway. This is now holding up a store submission because my functionality doesn't work on web.

Possibly related: #35

## Expected Behavior
During presenting mode `goToById` should change the currently visible slide.

## Current Behavior
Nothing happens

## Steps to Reproduce
1. Open a slideshow on PowerPoint web and start the slideshow
2. Run `Office.context.document.goToByIdAsync(Office.Index.Next,Office.GoToType.Index, () => {})` from an add-in or DevTools console.
3. Nothing happens.

## Live Example

See https://github.com/gordonmleigh/repro-officejs-875.

## Context
I'm trying to control the slideshow from the add-in, which works on desktop.

## Your Environment
* Platform: Office Online
* Host: PowerPoint
* Office version number: web
* Operating System: MacOS
* Browser (if using Office Online): Chrome 78

**Comment by donlvMSFT** [comment]
Hi @gordonmleigh , sorry for that the original issue #875 is closed by accident, probably because the tag is not proper set. Really sorry for that.
We already filed the bug 3809617 tracking this, and will come back if there is any progress.

**Comment by gordonmleigh** [comment]
@donlvMSFT we've had a submission to the store rejected because the functionality doesn't work in Office online. Have you any idea what the timescales will be on this fix?

Do we need to wait until this bug is fixed by your team before we can go ahead, or can we resubmit quoting your internal bug ID in the meantime?

**Comment by xiaochunms** [comment]
@gordonmleigh I am sorry for the inconvenience to bring to you. I am afraid that the submission may not success if the functionality doesn't work. There is a bug #3809617 filed to track the issue. The bug fix process may can be accelerated if you provide some data on how big the impact is. You may can drop me an email (xiliang@microsoft.com) if the data is sensitive.
